graalvm/src/main/java/com/example/GraalvmImageFactory.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.example;
+
+import ai.djl.modality.cv.BufferedImageFactory;
+import ai.djl.modality.cv.Image;
+import org.apache.commons.imaging.ImageFormat;
+import org.apache.commons.imaging.ImageFormats;
+import org.apache.commons.imaging.Imaging;
+import org.apache.commons.imaging.ImagingException;
+
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URL;
+import java.nio.file.Path;
+
+public class GraalvmImageFactory extends BufferedImageFactory {
```

Comment:
Why do we need to reimplement the BufferImageFactory?
is this the limitation of Graalvm?

Reply:
1, ImageIO doesn't work GraalVM, we have to use apache commons imaging instead
2. Apache commons imaging has many limitation, it doesn't support save JPG file, it has a bug loading some of JPG (only fixed in master branch, not released), so commons imaging is not a good candidate for default.